tests/core/editable/scrolls.js

```diff
@@ -0,0 +1,73 @@
+( function() {
+	'use strict';
+
+	bender.editors = {
+		divarea: {
```

Comment:
Inline editor should be added here too.

Reply:
I'm not sure about it. Is it possible to generate inline editor with scrolling editable inside our unit test? Or it require prepare special test suite?

Reply:
Aren't appropriate CSS styles (`overflow: auto` + small height) enough?